api.web.yaml

```diff
@@ -321,3 +320,8 @@ components:
       type: array
       items:
         $ref: "#/components/schemas/Person"
+
+    TrainingRecordKind:
+      type: string
+      enum:
+        - "nhsd"
```

Comment:
suggestions welcome on whether this should be ''nhsd' or 'training_kind_nhsd'

Reply:
- Although it is more verbose, I'd vote for `training_kind_nhsd` because implementations like `openapi.training_kind_nhsd` gives more context than `openapi.Nhsd`